src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/ExceptionsTests.cs

```diff
@@ -457,6 +457,288 @@ public async Task Exceptions_FilterIncludeDetailed(Architecture targetArchitectu
                 });
         }
 
+        [Theory]
+        [MemberData(nameof(ProfilerHelper.GetArchitecture), MemberType = typeof(ProfilerHelper))]
+        public async Task Exceptions_FilterIncludeBasic_Configuration(Architecture targetArchitecture)
+        {
+            await ScenarioRunner.SingleTarget(
+                _outputHelper,
+                _httpClientFactory,
+                DiagnosticPortConnectionMode.Listen,
+                FilteringExceptionsScenario,
+                appValidate: async (appRunner, apiClient) =>
+                {
+                    await GetExceptions(apiClient, appRunner, ExceptionFormat.PlainText);
+
+                    ValidateSingleExceptionText(
+                        SystemInvalidOperationException,
+                        ExceptionMessage,
+                        FrameClassName,
+                        FrameMethodName,
+                        new() { FrameParameterType, FrameParameterType });
```

Comment:
Please use type name

Reply:
This is a similar ask as https://github.com/dotnet/dotnet-monitor/pull/5217#discussion_r1319172815. It's not obvious what `new ()` is creating here.

Reply:
Oh, I totally interpreted that the wrong way *face palms* - I'll add that in the next revision.